app/src/main/java/doit/study/droid/data/local/entity/Question.kt

```diff
@@ -0,0 +1,24 @@
+package doit.study.droid.data.local.entity
+
+import androidx.annotation.NonNull
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import java.util.*
+
+@Entity
+data class Question (
+        @PrimaryKey
+        @NonNull
+        // Read only
+        val id: Int,
+        val text: String,
+        val wrong: List<String>,
+        val right: List<String>,
+        val docLink: String,
+        // Read/Write
+        val wrongCounter: Int = 0,
+        val rightCounter: Int = 0,
+        val consecutiveRightCounter: Int = 0,
+        val lastViewedAt: Long = Date().time,
+        val studiedAt: Long = 0
+)
```

Comment:
I do not use ProGuard obfuscation even in the release build, yet.
But it's a good catch and it can be revealed only after some release iterations, thx
Will be addressed here https://github.com/mgolokhov/dodroid/issues/140